addons/account_bank_statement_import/account_bank_statement_import.py

```diff
@@ -117,7 +119,7 @@ def _check_parsed_data(self, stmts_vals):
         if no_st_line:
             raise UserError(_('This file doesn\'t contain any transaction.'))
 
-    def _find_additional_data(self, currency_code, account_number):
+    def _find_additional_data(self, currency_code, account_number, bank_bic):
```

Comment:
as the 9.0 is stable now, it's better not to change the signature (too much), so i'd rather see a kwarg for bank_bic with default value